server/src/controllers/ScheduleController.js

```diff
@@ -0,0 +1,88 @@
+const ScheduleServices = require('../services/ScheduleServices');
+
+const ScheduleController = {
+  /**
+   * Get an array of schedule objects for a frosh group
+   * @param {Object} req
+   * @param {Object} res
+   * @param {Function} next
+   * @return {Object[]}
+   */
+  async getGroupSchedule(req, res, next) {
+    const { froshGroup } = req.body;
+
+    res.status(200).send({ groupSchedule: await ScheduleServices.getGroupSchedule(froshGroup) });
```

Comment:
i don't like this pattern you've been using of calling a function directly within the `.send` method - too many chances for things to go wrong. Please refactor anything like this out into two distinct calls: one to get the result of the awaited function, and another to send it in the `res.send()`.